modules/price/commerce_price.module

```diff
@@ -26,7 +26,9 @@ function commerce_price_theme() {
  * Implements hook_ENTITY_TYPE_insert() for 'configurable_language'.
  */
 function commerce_price_configurable_language_insert(ConfigurableLanguage $language) {
-  // Allow the importer to import any found translations for the new language.
-  $importer = \Drupal::service('commerce_price.currency_importer');
-  $importer->importTranslations([$language->getId()]);
+  if (!\Drupal::isConfigSyncing()) {
```

Comment:
(Just because it looks like a bug fix, which could go into its own commit and fix. Unless triggered here.)